common/artifact-manager/src/main/java/com/vmware/pscoe/iac/artifact/rest/RestClientVro.java

```diff
@@ -191,23 +200,42 @@ public void doWithRequest(ClientHttpRequest request) throws IOException {
             @Override
             public Void extractData(ClientHttpResponse response) throws IOException {
                 if (!dryrun) {
-                    Path path = Paths.get(pkg.getFilesystemPath());
+					Path path = Paths.get(pkg.getFilesystemPath());
+
+					System.out.println("About to copy the file");
```

Comment:
Do we have a logger class instance to cover these messages?

Reply:
@ivo-kotev there is a static logger in the RestClientVro used by other pieces of code as well